docs/ddl-of-carbondata.md

```diff
@@ -638,7 +638,12 @@ CarbonData DDL statements are documented here,which includes:
 
 
 ## CREATE DATABASE 
-  This function creates a new database. By default the database is created in Carbon store location, but you can also specify custom location.
+  This function creates a new database. By default the database is created in location 'spark.sql.warehouse.dir', but you can also specify custom location by configure 'spark.sql.warehouse.dir', the configuration 'carbon.storelocation' has been deprecated.
```

Comment:
```suggestion
  This function creates a new database. By default the database is created in location 'spark.sql.warehouse.dir', but you can also specify custom location by configuring 'spark.sql.warehouse.dir', the configuration 'carbon.storelocation' has been deprecated.
``

Reply:
I have modified code according to your suggestion